lib/CMakeLists.txt

```diff
@@ -1,3 +1,13 @@
+
+# This definition is due to older buggy C++11 cstdint implementations which do
```

Comment:
I would not say it was buggy. It works exactly as it was described. That it might look unreasonable is another question. :)